components/checkbox/src/index.js

```diff
@@ -88,23 +88,25 @@ const StyledLabel = styled('span')({
 });
 
 const Checkbox = ({
-  children, className, inline, ...input
+  children, inline, disabled, defaultChecked, ...props
```

Comment:
i'm not so sure about these changes

the intent of the `checkbox` component is that it works the same as `<input type="checkbox">`...